ppg-server/ppg-server.spec

```diff
@@ -0,0 +1,30 @@
+%global sname   percona-ppg-server
+%global pgmajorversion 14
+%global version 4
+
+Summary:        Base selection of PostgreSQL%{pgmajorversion} components
+Name:           %{sname}-%{pgmajorversion}
```

Comment:
The package name should include the version without the hyphen. This would otherwise would replace a version 13 meta package with a version 14.

Reply:
Fixed that